### ISSUE TITLE
Chapter 34 register recovery is incomplete

### ISSUE THREAD

**Issue by madgen** [sjl/learnvimscriptthehardway#96]
On chapter 34, you teach the how to not pollute the registers and recover the `"@` register.

This is correct, but it doesn't work if `clipboard` is set to `unnamed`. In that case `"+` and `"*` registers are used instead, so they should also be recovered.

Thanks a lot by the way, I learn a lot from the online version and will definitely buy it as soon as I finish reading it.

If you start accepting PRs again let me know and I'll happily create one.

**Comment by dahn-d** [comment]
this repo is abandoned. look at history

**Comment by madgen** [comment]
Yes, I know but I also know if someone else runs into the same problem issues would be one of the obvious places to look.

**Comment by mskadu** [comment]
Hi @madgen - Perhaps you'd like to contribute to [my fork](https://github.com/mskadu/learnvimscriptthehardway) of this project?
